Makefile.js

```diff
@@ -94,10 +94,10 @@ function getTestFilePatterns() {
         return test("-d", testLibPath + pathToCheck);
     }).reduce(function(initialValue, currentValues) {
         if (currentValues !== "rules") {
-            initialValue.push(`${testLibPath + currentValues}/**/*.js`);
+            initialValue.push(`"${testLibPath + currentValues}/**/*.js"`);
         }
         return initialValue;
-    }, [`${testLibPath}rules/**/*.js`, `${testLibPath}*.js`, `${testTemplatesPath}*.js`, `${testBinPath}**/*.js`]).join(" ");
+    }, [`"${testLibPath}rules/**/*.js"`, `${testLibPath}*.js`, `${testTemplatesPath}*.js`, `"${testBinPath}**/*.js"`]).join(" ");
```

Comment:
this might be safer as `'"' + paths.join('" "') + '"'` or the equivalent, so that nobody has to remember later which paths need quoting?